src/main/java/com/monumental/models/Model.java

```diff
@@ -1,15 +1,11 @@
 package com.monumental.models;
 
-import org.hibernate.annotations.CreationTimestamp;
-import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.jpa.domain.AbstractAuditable;
 
 import javax.persistence.*;
-import javax.validation.constraints.NotNull;
-import javax.validation.constraints.Null;
-import java.util.Date;
 
 @MappedSuperclass
-public abstract class Model {
+public abstract class Model extends AbstractAuditable<User, Integer> {
```

Comment:
Why did no one tell me this existed in 343 😢

Reply:
Also in order to use this I had to make an empty User model

Reply:
That's dumb

Reply:
Well, one day it won't be